scripts/index.js

```diff
@@ -0,0 +1,111 @@
+// Variable assignment
+let currentTemp = 65;
+
+const skyImages = {
+    sunny: ["https://media.giphy.com/media/lyVNcb1n5Ob0Q/giphy.gif", "The sun shines directly into the camera lens through tall evergreens. The camera angle moves slightly."],
+    cloudy: ["https://media.giphy.com/media/lOkbL3MJnEtHi/giphy.gif", "The sun shines through clouds racing across the sky."],
+    rainy: ["https://media.giphy.com/media/vLi3T5m3RH45y/giphy.gif", "A hyper realistic animation. of heavy rain bouncing off tiled patio"],
+    snowy: ["https://media.giphy.com/media/Fh3ezinVpi4yk/giphy.gif", "A hyper realiztic animation of heavy snow falling through evergreens against distant mountains."]
+};
+
+// temperature functions
+// increase temp helper function
+const addTemp = () => {
+    currentTemp += 1;
+    updateTemp(currentTemp);
+};
+// decrease temp helper function
+const subtractTemp = () => {
+    currentTemp -= 1;
+    updateTemp(currentTemp);
+};
+// Font changes for temp
+const changeTempColor = (currentTemp) => {
+    const temperatureCountContainer = document.getElementById("temperatureCount");
+    let color = ''
+    if (currentTemp >= 80) {
+        color = 'red';
+    } else if (currentTemp >= 70) {
+        color= 'orange';
+    } else if (currentTemp >= 60) {
+        color = 'yellow';
+    } else if (currentTemp >= 50) {
+        color = 'green';
+    } else if (currentTemp <= 49) {
+        color = 'blue'
+    };
+    document.getElementById("temperatureCount").style.color = color;
```

Comment:
Consider using classes and than adding this style to the css.
```suggestion
    document.getElementById("temperatureCount").className = color;
```

```
#styles.css

.blue {
     color: blue
}
```